cardano-cli/cardano-cli.cabal

```diff
@@ -216,33 +228,31 @@ test-suite cardano-cli-test
                         Test.Cli.Pioneers.Exercise6
                         Test.Cli.Pipes
                         Test.Cli.Shelley.Run.Query
-                        Test.OptParse
 
   ghc-options:          -threaded -rtsopts -with-rtsopts=-N -with-rtsopts=-T
 
 test-suite cardano-cli-golden
   import:               project-config
 
-  hs-source-dirs:       test
+  hs-source-dirs:       golden
   main-is:              cardano-cli-golden.hs
   type:                 exitcode-stdio-1.0
 
-  build-depends:        aeson             >= 1.5.6.0
+  build-depends:        aeson >= 1.5.6.0
                       , base16-bytestring
                       , bytestring
                       , cardano-api
+                      , cardano-api:gen
```

Comment:
are you sure this syntax is supported? we're using cabal-version 3.0 here, and [this is from 3.4](https://cabal.readthedocs.io/en/stable/cabal-package.html#sublibs)

Reply:
It passes CI 😁